cluster-scope/overlays/nerc-ocp-infra/machineconfigs/refresh-storage-interface.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: refresh-storage-interface
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+  systemd:
+    units:
+      - name: refresh-storage-interface.timer
+        enabled: true
+        contents: |
+          [Timer]
+          OnBootSec=3d
+          OnUnitActiveSec=3d
+          RandomizedDelaySec=4h
+
+          [Install]
+          WantedBy=timers.target
+      - name: refresh-storage-interface.service
```

Comment:
I guess the service doesn't need to be enabled when there's a corresponding timer for it?

Reply:
Correct. According to documentation:
> For each timer file, a matching unit file must exist, describing the unit to activate when the timer elapses. By default, a service by the same name as the timer (except for the suffix) is activated.